src/encoded/static/components/cart/util.js

```diff
@@ -67,11 +76,34 @@ export const isAllowedElementsPossible = (resultFilters) => {
 
 /**
  * Return a new array containing the merged contents of two carts with no duplicates. Contains odd
- * ES6 syntax to merge, clone, and dedupe arrays in one operation.
+ * ES6 syntax to merge, clone, and de-dupe arrays in one operation.
  * https://stackoverflow.com/questions/1584370/how-to-merge-two-arrays-in-javascript-and-de-duplicate-items#answer-38940354
  * @param {array} cart0AtIds Array of @ids in one cart
  * @param {array} cart1AtIds Array of @ids to merge with `cart0AtIds`
  */
 export const mergeCarts = (cart0AtIds, cart1AtIds) => (
     [...new Set([...cart0AtIds, ...cart1AtIds])]
 );
+
+
+/**
+ * Determine whether the given search results is for a cart search or not.
+ * @param {object} context Search-results object
+ *
+ * @return {boolean} True if search results are for a cart search.
+ */
+const cartSearchPaths = ['/cart-search/', '/cart-matrix/', '/cart-report/'];
+export const getIsCartSearch = (context) => {
```

Comment:
should this method be called "IsCartSearchable"? In theory you can even return "cartSearchPaths.includes" if you are into functional programming.

Do you need a null check to do defensive programming? This method is exported and there is no telling who passes what to it in the future.

Reply:
Calling the method `isCartSearchable` makes it impossible to call its return value variable `isCartSearchable`.